lightning/src/ln/channelmanager.rs

```diff
@@ -4739,7 +4793,7 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 					break Ok((raa_updates.accepted_htlcs, raa_updates.failed_htlcs,
 							raa_updates.finalized_claimed_htlcs,
 							chan.get().get_short_channel_id()
-								.expect("RAA should only work on a short-id-available channel"),
+								.unwrap_or(chan.get().outbound_scid_alias()),
```

Comment:
If we're going to  make this change in this commit, should we also update the `get_short_channel_id()` unwrap in `handle_chan_restoration_locked`?

Reply:
Oh, yea, lets just move this hunk to a later PR.